launcher/ui/pages/instance/LogPage.cpp

```diff
@@ -277,28 +278,21 @@ void LogPage::on_btnPaste_clicked()
     //FIXME: turn this into a proper task and move the upload logic out of GuiUtil!
     m_model->append(
         MessageLevel::Launcher,
-        QString("%2: Log upload triggered at: %1").arg(
-            QDateTime::currentDateTime().toString(Qt::RFC2822Date),
-            BuildConfig.LAUNCHER_DISPLAYNAME
+        QString("Log upload triggered at: %1").arg(
+            QDateTime::currentDateTime().toString(Qt::RFC2822Date)
         )
     );
-    auto url = GuiUtil::uploadPaste(m_model->toPlainText(), this);
-    if(!url.isEmpty())
+    auto url = GuiUtil::uploadPaste(tr("Minecraft Log"), m_model->toPlainText(), this);
+    if(url == "canceled")
     {
-        m_model->append(
-            MessageLevel::Launcher,
-            QString("%2: Log uploaded to: %1").arg(
-                url,
-                BuildConfig.LAUNCHER_DISPLAYNAME
-            )
-        );
+        m_model->append(MessageLevel::Error, QString("Log upload canceled"));
     }
-    else
+    else if(!url.isEmpty())
     {
-        m_model->append(
-            MessageLevel::Error,
-            QString("%1: Log upload failed!").arg(BuildConfig.LAUNCHER_DISPLAYNAME)
-        );
+        m_model->append(MessageLevel::Launcher, QString("Log uploaded to: %1").arg(url));
+    }
+    else {
+        m_model->append(MessageLevel::Error, QString("Log upload failed!"));
```

Comment:
As GitHub doesn't allow me to make a suggestion here, I am just going to dump the code here:

```cpp
    auto url = GuiUtil::uploadPaste(tr("Minecraft Log"), m_model->toPlainText(), this);
    if(url.isNull())
    {
        m_model->append(MessageLevel::Error, QString("Log upload failed!"));
    }
    else if(url.isEmpty())
    {
        m_model->append(MessageLevel::Error, QString("Log upload canceled"));
    }
    else {
        m_model->append(MessageLevel::Launcher, QString("Log uploaded to: %1").arg(url));
    }

```